setup.py

```diff
@@ -343,7 +344,13 @@ def run(self):
       author='Intel Corporation',
       packages=find_packages(),
       package_data={'sdc.tests': ['*.bz2'], },
-      install_requires=['numba'],
+      install_requires=[
+          'scipy',
+          'numpy>=1.16',
+          'pandas==0.25.3',
+          'pyarrow==0.15.1',
+          'numba==0.48'
```

Comment:
Can't we get it from requirements.txt somehow?

Reply:
Requirements is just a file which can be used for packages installation
Main requirements are set upped here